packages/devtools_app/lib/src/shared/analytics/constants.dart

```diff
@@ -245,6 +250,14 @@ class MemoryEvent {
   static const diffSnapshotFilterReset = 'diffSnapshotFilterReset';
 
   static const browseRefLimit = 'browseRefLimit';
+
+  static const dropOneLiveVariable = 'dropOneLiveVariable';
+  static const dropOneStaticVariable = 'dropOneStaticVariable';
+  static String dropAllLiveToConsole({
+    required bool includeSubclasses,
+    required bool includeImplementers,
+  }) =>
+      'dropAllVariables${includeSubclasses ? 'Subclasses' : ''}${includeImplementers ? 'Imlementers' : ''}';
```

Comment:
nit: add an underscore before Subclasses and Implementers so there is some separation